### DISCUSSION TITLE
Cant find Integration in HACS

### DISCUSSION THREAD

**Discussion by WolfgangLandauer** [alandtse/alexa_media_player#2705] [Q&A]
Hi! I am not able to find the integration withn HACS/Add On. Do i need to add a repositiory? If yes, can you tell me the link please

**Comment by jo8be** [comment]
I just searched in HACS for "Alexa" and found it instantly...?

**Comment by danielbrunt57** [comment]
> Hi! I am not able to find the integration withn HACS/Add On. Do i need to add a repositiory? If yes, can you tell me the link please

You do not need to add it as a custom repository as it is already in the HACS Community Store.

![image](https://github.com/user-attachments/assets/49ec910e-450e-47f1-9c5a-5196cd243d3d)

If HACS is not showing it to you, you **_might_** have to re & re HACS.
A re & re does not lose any of your existing installed repositories.
